src/autofill-utils.js

```diff
@@ -567,23 +567,16 @@ function getActiveElement(root = document) {
 }
 
 /**
- * Takes a root element and tries to find visible elements first, and if it fails, it tries to find shadow elements
+ * Takes a root element and tries to find elements in shadow DOMs that match the selector
```

Comment:
As mentioned above, I'm a little unclear on why we decided to change this abstraction.